src/items/FileItem.stories.tsx

```diff
@@ -81,7 +81,7 @@ export const ImageSVG: Story = {
       type: 'file',
       description: 'my svg description',
       path: 'item-path',
-      settings: {},
+      settings: { altText: 'my svg alt text' },
```

Comment:
I guess this will change if you move to extra? @spaenleh

Reply:
Yes thank you need to update that.